modules/radio/css/README.md

```diff
@@ -0,0 +1,39 @@
+# Canvas Kit Radio
+
+Radio styles
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-css
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-css-radio
+```
+
+Add your `node_modules` directory to your SASS `includePaths`. You will then be able to import
+`index.scss`.
+
+```scss
+@import '~@workday/canvas-kit-css-radio/index.scss';
+```
+
+## Usage
+
+```html
+<div className="wdc-form-radio">
+  <input type="radio" id="radio" />
+  <label htmlFor="radio">E-mail</label>
```

Comment:
Looks like we still use React-specific attributes (`className`, `htmlFor`, etc.) in several CSS READMEs. Radio is just one example, but I've seen it in other module READMEs as well.